Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Magazines/light_rifle.yml

```diff
@@ -36,14 +36,17 @@
 # Magazines
 - type: entity
   id: MagazineLightRifleBox
-  name: "L6 SAW magazine box (.30 rifle)"
+  name: "L6 SAW magazine box (.20 rifle)"
   parent: BaseMagazineLightRifle
   components:
   - type: Tag
     tags:
       - MagazineLightRifleBox
   - type: BallisticAmmoProvider
-    proto: CartridgeLightRifle
+    whitelist:
+      tags:
+        - CartridgeRifle
+    proto: CartridgeRifle
```

Comment:
i'm not entirely sure about the whole .30 -> .20 thing with this pr, but i'm also not a gun nerd so i dunno. this is probably a good thing?